From: Carlos Garnacho <carlosg@gnome.org>
Date: Sat, 28 May 2016 20:44:19 +0200
Subject: docs: Do not delete xml directory when redoing docs

We're competing with gtk-doc.make here, which may turn out in
failures when doing mkdir on an already created directory, or
failures in writing xml/gtkdocentities.ent because we deleted
the directory under its feet.

So just do the same, use MKDIR_P to ensure it won't fail if
the directory is already there, and don't bother deleting it
as it's handled in gtk-doc.make. The ontology docs helper app
will rewrite the xml files anyway.
---
 docs/reference/ontology/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/reference/ontology/Makefile.am b/docs/reference/ontology/Makefile.am
index a7eeafa..1c6a2c2 100644
--- a/docs/reference/ontology/Makefile.am
+++ b/docs/reference/ontology/Makefile.am
@@ -12,7 +12,7 @@ ONTOLOGY_EXPLANATIONS =                                \
 
 # Generation of the ontology XML files.
 gen-doc.stamp: $(ONTOLOGY_EXPLANATIONS)
-	rm -rf xml && mkdir xml
+	$(MKDIR_P) xml
 	$(top_srcdir)/docs/tools/gen-doc.sh $(top_builddir)/docs/tools/ttl2sgml $(top_builddir)/docs/tools/ttlresource2sgml $(top_srcdir)/src/ontologies $(top_srcdir)/docs/ontologies xml/
 	$(AM_V_GEN) touch $@
 
